assets/styles/index.scss

```diff
@@ -3219,14 +3219,22 @@ a.wire-articles__versions {
 
 .blue-circle {
     background-color: $main-blue;
-    border: 1px solid $main-blue;
+    border: 1px solid #0600ec;
     box-shadow: 0px 0 0 2px #dedede;
     border-radius: 50%;
     height: 8px;
     width: 8px;
     bottom: -1px;
-    right: -4px;
+    right: 19px;
     position: absolute;
+
+    &--pale {
+        background-color: #00adec4f;
+    }
+
+    &--right {
```

Comment:
The blue circle does not overlay the icon properly in mobile phone views (it is overlaying the dashed separator instead of the icon border)